train_hierarchialXC.py

```diff
@@ -133,6 +129,7 @@ def generate_encodings(data_loader, ae):
     train_data_loader = torch.utils.data.DataLoader(train_loader, batch_size=args.batch_size,
                                                     shuffle=True, **loader_kwargs)
 
+    opt_options = yaml.load(open(args.optimizer_cfg))
```

Comment:
Assign the default as `None`, and check for it here. That is, if `args.optimizer_cfg` is `None`, then raise a `ValueError` telling that the config was not specified.